scripts/irods/test/test_iadmin.py

```diff
@@ -1708,6 +1708,36 @@ def test_iadmin_exits_nonzero_on_nonexistent_command__issue_7734(self):
         ec, _, _ = self.admin.assert_icommand(['iadmin', 'nonexistentcommand'], 'STDERR');
         self.assertNotEqual(ec, 0)
 
+    def test_iadmin_rmuser_removes_associated_data__issue_7778(self):
+        test_user_name = 'test_user_issue_7778'
+        test_group_name = 'test_group_issue_7778'
+        test_collection_name = 'test_collection_issue_7778'
+        try:
+            self.admin.assert_icommand(['imkdir', test_collection_name])
+            self.admin.assert_icommand(['iadmin', 'mkgroup', test_group_name])
+            self.admin.assert_icommand(['iadmin', 'mkuser', test_user_name, 'rodsuser'])
+            self.admin.assert_icommand(['iadmin', 'atg', test_group_name, test_user_name])
+            self.admin.assert_icommand(['ichmod', 'read', test_group_name, test_collection_name])
+            self.admin.assert_icommand(['ichmod', 'write', test_user_name, test_collection_name])
+
+            # Collect IDs of user and group
+            _, out, _ = self.admin.assert_icommand(['iquest', '%s', "SELECT USER_ID WHERE USER_NAME = '%s' || = '%s'" % (test_user_name, test_group_name) ], 'STDOUT_MULTILINE', r'^[0-9]+$', use_regex=True)
+
+            # Create iquest query string for the IDs-- we don't care about order, we just want both
+            query_string = "= '%s' || = '%s'" % tuple(out.strip().split('\n'))
+
+            # Should see the permissions as expected
+            self.admin.assert_icommand(['iquest', "SELECT COLL_ACCESS_USER_ID, COLL_ACCESS_NAME WHERE COLL_NAME = '%s' AND COLL_ACCESS_USER_ID %s" % (os.path.join(self.admin.session_collection, test_collection_name), query_string)], 'STDOUT', ['COLL_ACCESS_NAME = read_object', 'COLL_ACCESS_NAME = modify_object'])
+
+            self.admin.assert_icommand(['iadmin', 'rmuser', test_user_name])
+            self.admin.assert_icommand(['iadmin', 'rmgroup', test_group_name])
```

Comment:
Even though I know the removal of ACLs is tied to the user ID... It feels like it may be worth checking ACLs in between removal of the user and the group to make sure the user's ACLs are gone but the group's ACLs remain. Thoughts?

Also, let's make sure the data itself remains (the name of the test concerns me :p).